lib/puppet/provider/netapp_quota/cmode.rb

```diff
@@ -186,6 +186,7 @@ def flush
     if status('volume', volume).child_get_string('status') == 'on'
       if @need_restart
         qoff 'volume', volume
+        sleep 5 
```

Comment:
This is just hard coded... is there another command that would be blocking for any short or long amount of time to make this better? Like doing a quota off, list/status, and then an on?